templates/_macros/vf_cta-section.jinja

```diff
@@ -0,0 +1,105 @@
+# All Params
+  # title_text: H2 title text
+  # link_text: H2 Link text
+  # link_href: href for the link text
+  # layout: Layout type of cta section. Options are "cta-default-100", "cta-default-25-75", "cta-block-100", "cta-block-25-75". Default is "cta-default-100".
```

Comment:
The layout param here seems to combine two separate pieces of information: `default` or `block`, and `25-75` or `100`. Might this be better suited to two parameters, like `variant` for `default`/`block`, and `layout` for `25-75`/`100`? 

if you think this is OK, it should also follow the convention from the Hero pattern that accepts hyphen-separated layouts and slash-separated layouts: `50/50` and `50-50` leverage the same layout. See [example](https://github.com/canonical/vanilla-framework/blob/45b3421ba5aead447f3c0b6717713072b95d0292/templates/_macros/vf_hero.jinja#L29-L33)